src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java

```diff
@@ -98,6 +98,7 @@ private SlimefunItems() {}
 	
 	/*		 Food 		*/
 	public static final ItemStack FORTUNE_COOKIE = new CustomItem(Material.COOKIE, "&6Fortune Cookie", "", "&a&oTells you stuff about your Future :o");
+	public static final ItemStack DIET_COOKIE = new CustomItem(Material.COOKIE, "&6Flying Cookie", "", "&a&oA boring diet cookie.");
```

Comment:
Better idea:

Keep the name "&6Diet Cookie" and change the lore to "&a&oA cookie so light, it makes you levitate!"

Reply:
But I wanted the levitation effect a bit hidden. To create a surprising effect on the player

Reply:
Well, "Flying Cookie" is a bit misleading though, players could expect they will fly and then just levitate.

Reply:
How about this lore?

"&eA very &olightweight &r&ecookie"

And the name "&6Diet Cookie"

Reply:
that sounds good

P.S
I originally made the cookie with the name FLYING_COOKIE but then I changed it to DIET_COOKIE I guess I just forgot to change the name part lol